components/automate-ui/src/app/pages/+compliance/+reporting/+reporting-node/reporting-node.component.ts

```diff
@@ -205,4 +210,44 @@ export class ReportingNodeComponent implements OnInit, OnDestroy {
         this.activeReport = Object.assign(report, data);
       });
   }
+  
+  toggleDownloadDropdown() {
+    this.downloadOptsVisible = !this.downloadOptsVisible;
+  }
+  hideDownloadStatus() {
+    this.downloadStatusVisible = false;
+    this.downloadInProgress = false;
+    this.downloadFailed = false;
+  }
+
+  showDownloadStatus() {
+    this.downloadStatusVisible = true;
+    this.downloadInProgress = true;
+    this.downloadFailed = false;
+  }
+
+  onDownloadNodeReport(format) {
+    this.downloadOptsVisible = false;
+    const id: string = this.route.snapshot.params['id'];
+    const reportQuery = this.reportQueryService.getReportQuery();
+    reportQuery.filters = reportQuery.filters.concat([{type: {name: 'node_id'}, value: {id}}]);
+    const filename = `${reportQuery.endDate.format('YYYY-M-D')}.${format}`;
```

Comment:
This is confusing because the parameter name `format` is also a method name. Consider changing the parameter name here. (Also in stats.service.ts::downloadNodeReport)